setupmeta/model.py

```diff
@@ -310,6 +310,16 @@ def get_pip():
 
         return parse_requirements, PipSession
 
+    except ImportError:
+        pass
+
+    try:
+        # pip >= 19.3
+        from pip._internal.req import parse_requirements
+        from pip._internal.network.session import PipSession
+
+        return parse_requirements, PipSession
+
```

Comment:
Should try newest first I think.

Reply:
Very good point, I updated code to do that now: https://github.com/zsimic/setupmeta/commit/7690997f83499eee78267286695d0b5109382666

Following shortly with release